### PR TITLE
Add option to replicaset filter

### PR DESCRIPTION
**Pull request by AlexandrLitkevich**
What has been done? Why? What problem is being solved?

I didn't forget about

- [ ] Tests
- [x] Changelog
- [ ] Documentation

Close #???

### CODE REVIEW

File: webui/src/models/cluster/server-list/selectors.ts [hunk 1]
```diff
@@ -31,6 +31,9 @@ export const serverRo = (server: ServerListServer): boolean | undefined => serve
 export const replicasetServerRo = (server: ServerListReplicasetServer): boolean | undefined =>
   server.boxinfo?.general?.ro;
 
+export const replicasetServerIsLeader = (server: ServerListReplicasetServer): string | undefined =>
```
[the remark this answers is not in the document]
Comment (AlexandrLitkevich) [reply webui/src/models/cluster/server-list/selectors.ts]: Done

File: webui/src/pages/Cluster/components/ReplicasetFilter/ReplicasetFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React from 'react';
+import { cx } from '@emotion/css';
+// prettier-ignore
+// @ts-ignore
+import { Button, DropdownDivider, DropdownItem, IconChevron, IconSearch, Input, withDropdown, } from '@tarantool.io/ui-kit';
+
+import { styles } from 'src/pages/Cluster/components/ReplicasetFilter/ReplicasetFilter.styles';
```
[the remark this answers is not in the document]
Comment (AlexandrLitkevich) [reply webui/src/pages/Cluster/components/ReplicasetFilter/ReplicasetFilter.tsx]: Done

**Review by usenko-timur — APPROVED** [review]

**Comment by opomuc** [comment]
а этот тикет реально сделали или просто закрыли?

**Comment by filonenko-mikhail** [comment]
переделываем

**Pull request merged**